Test/Web/FunctionalTests/TestApps/AspNetDiagnostics/ApplicationInsights.config

```diff
@@ -42,9 +42,9 @@
     <Add Type="Microsoft.ApplicationInsights.WindowsServer.UnobservedExceptionTelemetryModule, Microsoft.AI.WindowsServer"/>
     <Add Type="Microsoft.ApplicationInsights.WindowsServer.FirstChanceExceptionStatisticsTelemetryModule, Microsoft.AI.WindowsServer"/>
   </TelemetryModules>
-  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel">
+  <TelemetryChannel Type="Microsoft.ApplicationInsights.Channel.InMemoryChannel, Microsoft.ApplicationInsights">
     <EndpointAddress>http://localhost:4000/v2/track</EndpointAddress>
-    <MaxTelemetryBufferDelay>0:00:01</MaxTelemetryBufferDelay>
+    <DeveloperMode>true</DeveloperMode>
```

Comment:
if devmode is true, then we dont need flush at all right?

Reply:
this is Web, there is no flush there. But I guess in dependency tracking functional tests yes, we don't. need it.

In theory, it should with ServerTelemteryChannel isn't it?

Reply:
Yes devmode=true means we should be flushing instantly after every item automatically.